[IMP] Fleet: Update new vehicles (#13213)
odoo/documentation: content/applications/hr/fleet/new_vehicle.rst

:show-content:

===============
Adding vehicles
===============

Odoo's **Fleet** app manages all vehicles, and the accompanying documentation that comes with

> vehicle maintenance and drivers records.

Reviewer: ```suggestion
vehicle maintenance and driver records.
```

Upon opening the :menuselection:`Fleet` application, all vehicles are organized within the
:guilabel:`Vehicles` dashboard, which is the default dashboard. Each vehicle is displayed in its
corresponding Kanban stage, based on its status. The default stages are :guilabel:`New Request`,
:guilabel:`To Order`, :guilabel:`Registered`, and :guilabel:`Downgraded`.

Additionally, two settings affect the **Fleet** app and should be :ref:`reviewed and configured
<fleet/settings>` before adding vehicles.

.. _fleet/settings:

Settings
========

To access the settings menu, go to :menuselection:`Fleet app --> Configuration --> Settings`. Only
two settings need configuration: :guilabel:`End Date Contract Alert` and :guilabel:`New Vehicle
Request`.

End Date Contract Alert
-----------------------

> The :guilabel:`End Date Contract Alert` field how many days before the end of a vehicle contract an

Reviewer: ```suggestion
The :guilabel:`End Date Contract Alert` field shows how many days before the end of a vehicle contract an
```

> alert should be sent. The responsible people receive an email informing them a vehicle contract is

Reviewer: ```suggestion
alert should be sent. The responsible people receive an email informing them that a vehicle contract is
```

about to expire in the number of days defined in this field.

.. note::

>    To determine who the responsible person is for a contract, open an individual contract. The

Reviewer: ```suggestion
   To determine the responsible person for a contract, open an individual contract. The
```

   person listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of
   the contract is the person who will receive the alert.

   To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
   contracts appear in the list. Click on a :guilabel:`Contract` to view it.

   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
   Fleet` and clicking on an individual vehicle. On the vehicle form, click the

>    :guilabel:`Contracts` smart button at the top of the page. Ony contracts associated with the

Reviewer: ```suggestion
   :guilabel:`Contracts` smart button at the top of the page. Only contracts associated with the
```

   vehicle appear in the list. Click on an individual contract to open it. The
   :guilabel:`Responsible` person is listed on the contract.

.. image:: new_vehicle/fleet-settings.png
   :alt: Settings available for the Fleet application.

New Vehicle Request
-------------------

The :guilabel:`New Vehicle Request` field sets a limit on how many new vehicles are requested based
on fleet availability. An employee filling out the salary configurator form (after being offered a
position), will *not* be able to request a new car if the number of existing cars is greater than
the number specified in the :guilabel:`New Vehicle Request` field. Enter the specific number limit

> for existing available cars in this field.

Reviewer: ```suggestion
for the existing available cars in this field.
```

.. example::
   If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
   available, an employee would not be able to request a new vehicle. If there are only 10 cars
   available, then the employee would be able to request a new vehicle.

.. note::

>    This settings option **only** appears if the :guilabel:`Belgian-Payroll-Fleet` module is

Reviewer: Did we confirm this is only with the Belgian localization? I know we found that additional module needed to be installed, but is it ONLY with the Belgian one?

Author: I am checking with Emilio about this- this one I am 99.9% sure is still correct. The other doc, I am waiting to hear back on. Mostly because we did more localizations this year- but only a few, and I don't know alllll the changes associated with them, or what else they make appear.

   installed for the Belgian localization.

Add a vehicle
=============

To add a new vehicle to the fleet from the :guilabel:`Vehicles` dashboard, click the :guilabel:`New`
button in the top-left corner, and a blank vehicle form loads. Then, proceed to enter the vehicle
information on the vehicle form.

.. _fleet/new_vehicle/vehicle-form:

Vehicle form fields
===================

- :guilabel:`Model`: using the drop-dpown menu, select the vehicle's model. Once a model is
  selected, additional fields may appear on the form. If the model is not listed, type in the model
  name, and click either :guilabel:`Create "model"`, or :guilabel:`Create and edit...` to
  :ref:`create a new model and edit the model details <fleet/add-model>`.

> - :guilabel:`License Plate`: enter the vehicle's license plate number.

Reviewer: full sentences after ":" should be capitalized

- :guilabel:`Tags`: select any tags from the drop-down menu, or type in a new tag. There is no limit
  on the amount of tags that can be selected.

.. image:: new_vehicle/model.png
   :alt: The new vehicle form, showing the model section.

.. note::
   The :guilabel:`Model` is the *only* required field on the new vehicle form. When a model is
   selected, other fields appear on the vehicle form, and relevant information auto-populates the
   fields that apply to the model. If some of the fields do not appear, this may indicate there is
   no model selected.

.. _fleet/new_vehicle/new-driver:

> DRIVER

Reviewer: Headings should be in sentence case (even if it is different from the UI)

------

This section of the vehicle form relates to the person who is currently driving the car, as well as
any plans for a change in the driver in the future, and when.

> - :guilabel:`Driver`: using the drop-down menu, select the driver, or type in a new driver and click

Reviewer: full sentences after ":" should be capitalized

:show-content:

===============
Adding vehicles
===============

Odoo's **Fleet** app manages all vehicles, and the accompanying documentation that comes with
vehicle maintenance and driver records.

Upon opening the :menuselection:`Fleet` application, all vehicles are organized within the
:guilabel:`Vehicles` dashboard, which is the default dashboard. Each vehicle is displayed in its
corresponding Kanban stage, based on its status. The default stages are :guilabel:`New Request`,
:guilabel:`To Order`, :guilabel:`Registered`, and :guilabel:`Downgraded`.

Additionally, two settings affect the **Fleet** app and should be :ref:`reviewed and configured

> <fleet/settings>` before adding vehicles.

Author: Now the following section has changed, I don't think we need this at all.

.. _fleet/settings:

Settings
========

> To access the settings menu, go to :menuselection:`Fleet app --> Configuration --> Settings`. Only

Author: SO much better! Plus, I added anchors to jump to those settings =)

two settings need configuration: :guilabel:`End Date Contract Alert` and :guilabel:`New Vehicle
Request`.

End Date Contract Alert
-----------------------

The :guilabel:`End Date Contract Alert` field shows how many days before the end of a vehicle
contract an alert should be sent. The responsible people receive an email informing them that a
vehicle contract is about to expire in the number of days defined in this field.

.. note::
   To determine the responsible person is for a contract, open an individual contract. The person
   listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of the
   contract receives the alert.

   To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
   contracts appear in the list. Click on a :guilabel:`Contract` to view it.

   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
   Fleet` and clicking on an individual vehicle. On the vehicle form, click the
   :guilabel:`Contracts` smart button at the top of the page. Only contracts associated with the
   vehicle appear in the list. Click on an individual contract to open it. The

>    :guilabel:`Responsible` person is listed on the contract.

Author: Good idea - I added a "Responsible parties' section and linked to it in the part above.

.. image:: new_vehicle/fleet-settings.png
   :alt: Settings available for the Fleet application.

> New Vehicle Request

Author: Ahh, yes- this is a confusing section. Basically, if there's enough cars, you're not allowed to ask for a new one. They want to use all their current inventory before bloating their fleet. So this just ensures that you can only ask for a brand new car, if there aren't enough cars lying around, waiting for drivers.

-------------------

The :guilabel:`New Vehicle Request` field limits how many new vehicles are requested based on fleet

> availability. An employee filling out the salary configurator form (after being offered a position),

Author: Ahh, ok - this is a good idea! Ive never seen a section be country-sppecific, so Ihad a hard time thinking about how to present it. I like that it says BELGIUM right in the leading! I just rephrased it, since it sounds like the limit is on the fleet vehicles in total (sort of)- the limit is on the AVAILABLE fleet vehicles. So if there are 5 available cars, and the limit is 10 - I can requesst a new car. BUT if there are 15 cars without drivers sitting in the lot, I can't request a new car. Doesn't matter if the fleet is 10 or 1,000 cars- it's based on the ones available.

will *not* be able to request a new car if the number of existing cars is greater than the number
specified in the :guilabel:`New Vehicle Request` field. Enter the specific number limit for the
existing available cars in this field.

.. example::
   If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
   available, an employee would not be able to request a new vehicle. If there are only 10 cars
   available, then the employee would be able to request a new vehicle.

.. note::

>    This settings option **only** appears if the :guilabel:`Belgian-Payroll-Fleet` module is

Author: I am going to leave it in because if they have the Belgian localization, and for some reason didn't install fleet originally, or something didn't install the module - they need to know this specific module is what makes that field appear (not JUST the Belgium localization). So I kept the heading, but am also keeping the note since this is a specific module and not just the localization. Also, it took a while for JERO and I to figure this out! And by "US" I mean JERO found it for me =D

.. image:: new_vehicle/model.png
   :alt: The new vehicle form, showing the model section.

.. note::
   The :guilabel:`Model` is the *only* required field on the new vehicle form. When a model is
   selected, other fields appear on the vehicle form, and relevant information auto-populates the
   fields that apply to the model. If some of the fields do not appear, this may indicate there is
   no model selected.

.. _fleet/new_vehicle/new-driver:

Driver
------

This section of the vehicle form relates to the person who is currently driving the car, as well as
any plans for a change in the driver in the future, and when.

- :guilabel:`Driver`: Using the drop-down menu, select the driver, or type in a new driver and click
  either :guilabel:`Create "driver"` or :guilabel:`Create and edit...` to :ref:`create a new driver,
  and edit the driver details <fleet/new_vehicle/add-driver>`.

  .. important::
     A driver does *not* have to be an employee. When creating a new driver, the driver is added to
     the **Fleet** app, *not* the **Employees** app.

     If the **Contacts** application is installed, the driver information is also stored there.

> - :guilabel:`Mobility Card`: If the selected driver has a mobility card listed on their employee

Author: Basically, a gas card or any card that they can use for Fleet expenses

  record in the **Employees** application, the mobility card number automatically appears in this
  field. If there is no mobility card listed, and one should be added, :ref:`edit the employee
  record <employees/hr-settings>` in the **Employees** application.

> - :guilabel:`Future Driver`: If the next driver for the vehicle is known, select the next driver

Author: It makes an 'Apply New Driver' button appear on the vehicle, so when it's time to change drivers, you just click the button and everything is switched! =) I added it.

> :show-content:

Reviewer 2: not sure if this line is necessary?

Author: I wasn't sure either =D I was looking at previous/other docs to see what was needed. I'll remove it, and see if it breaks anything!

.. _fleet/settings:

Settings
========

Before registering any vehicles, review two **Fleet** settings that directly affect day-to-day
operations and employee benefits:

- :ref:`End Date Contract Alert <fleet/end-contract>` — emails the contract's *Responsible* person
  a set number of days before the vehicle agreement expires, so renewals or returns aren't missed.
- :ref:`New Vehicle Request <fleet/request>` — (Belgian Payroll localization only) blocks employees
  from requesting a new company car through the salary configurator, once the available vehicles
  already exceeds the defined limit, helping control benefits costs.

To access the settings menu, go to :menuselection:`Fleet app --> Configuration --> Settings`.

.. _fleet/end-contract:

End Date Contract Alert
-----------------------

The :guilabel:`End Date Contract Alert` field shows how many days before the end of a vehicle
contract an alert should be sent. The :ref:`responsible parties <fleet/responsible>` receive
an email informing them that a vehicle contract is about to expire in the number of days defined
in this field.

.. _fleet/responsible:

> Responsible parties

Reviewer 2: oooh i like how you're calling it responsible *parties* :partying_face:

~~~~~~~~~~~~~~~~~~~

To determine the responsible person is for a contract, open an individual contract. The person
listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of the
contract receives the alert.

> To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all

Reviewer 2: ```suggestion
To open a contract from a list of all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
```

> contracts appear in the list. Click on a :guilabel:`Contract` to view it.

Reviewer 2: ```suggestion
contracts appear in the list. Click on a :guilabel:`Contract` to open it. In the :guilabel:`Information` section of the contract, look under :guilabel:`Responsible` to see who receives expiration alerts.
```

> An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->

Reviewer 2: There's an abrupt jump between two different navigational instructions to both find the Responsible field. How about adding a sentence like "The responsible field is found <describe a place>" on line 51 before moving onto the next instruction?

Author: I believe this is covered by the edits you made later in this review?

> An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->

Reviewer 2: ```suggestion
An individual contract can also be open from a specific vehicle by navigating to :menuselection:`Fleet app --> Fleet -->
```

Fleet` and clicking on an individual vehicle. On the vehicle form, click the :guilabel:`Contracts`
smart button at the top of the page. Only contracts associated with the vehicle appear in the list.
Click on an individual contract to open it. The :guilabel:`Responsible` person is listed on the
contract.

.. image:: new_vehicle/fleet-settings.png

>    :alt: Settings available for the Fleet application.

Reviewer 2: Let's move this image up, before the "responsbile parties" section, since based on the order, the setting seems like it's talking about responsible parties.

.. _fleet/request:

New Vehicle Request (Belgian Payroll - Fleet)
---------------------------------------------

Enter the maximum fleet size that may be reached through the salary-configurator flow. When the
total number of available vehicles (vehicles without an assigned driver) is equal to or below
this limit, employees can request a new company car. Once the available vehicles exceeds the
limit, the request option is hidden.

.. example::

>    If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
>    available, an employee would not be able to request a new vehicle. If there are only 10 cars

Reviewer 2: ```suggestion
   If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
   available, employees cannot request a new car and must select from the 25 already available. If there are only 10 cars
```

- :guilabel:`Tags`: Select any tags from the drop-down menu, or type in a new tag. There is no limit
  on the amount of tags that can be selected.

.. image:: new_vehicle/model.png
   :alt: The new vehicle form, showing the model section.

.. note::
   The :guilabel:`Model` is the *only* required field on the new vehicle form. When a model is
   selected, other fields appear on the vehicle form, and relevant information auto-populates the
   fields that apply to the model. If some of the fields do not appear, this may indicate there is
   no model selected.

.. _fleet/new_vehicle/new-driver:

Driver
------

This section of the vehicle form relates to the person who is currently driving the car, as well as
any plans for a change in the driver in the future, and when.

- :guilabel:`Driver`: Using the drop-down menu, select the driver for the vehicle. If the driver
  is not listed, :ref:`create the new driver, and edit the driver details <fleet/new_vehicle/add-driver>`.

  .. important::
     A driver does *not* have to be an employee. When creating a new driver, the driver is added to
     the **Fleet** app, *not* the **Employees** app.

>      If the **Contacts** application is installed, the driver information is also stored there.

Reviewer 2: super minor thing but im wondering if we should change this to match "app" usage in the admonition block
```suggestion
     If the **Contacts** app is installed, the driver information is also stored there.
```

- :guilabel:`Mobility Card`: If the selected driver has a mobility card (such as a gas card) listed
  on their employee record in the **Employees** application, the mobility card number automatically
  appears in this field. If there is no mobility card listed, and one should be added, :ref:`edit the
  employee record <employees/hr-settings>` in the **Employees** application.
- :guilabel:`Future Driver`: If the next driver for the vehicle is known, select the next driver
  from the drop-down menu. Or, type in the next driver and click either :guilabel:`Create "future
  driver"` or :guilabel:`Create and edit...` to :ref:`create a new future driver, and edit the
  driver details <fleet/new_vehicle/add-driver>`.

  .. note::
     If this field is populated, a :guilabel:`Apply New Driver` button appears on the vehicle form.
     Click the :guilabel:`Apply New Driver` button to change the driver information.

- :guilabel:`Plan To Change Car`: If the current driver set for this vehicle plans to change their
  vehicle - either because they are waiting on a new vehicle that is being ordered, or this is a

>   temporary vehicle assignment, and they know which vehicle they are driving next, then tick this box.

Reviewer 2: we can shorten this long sentence for clarity. How about

Tick this box when the current driver already knows they’ll switch to another vehicle—whether they’re awaiting an ordered car or using this one only temporarily.

Driver` field, then click :guilabel:`Create and edit...`. A :guilabel:`Create Driver` or
:guilabel:`Create Future Driver` form appears, depending on which field initiated the form.

Both the :guilabel:`Create Driver` and :guilabel:`Create Future Driver` forms are identical, and are
stored in the **Contacts** app. :doc:`Configure the new contact <../../essentials/contacts>`, then
click :guilabel:`Save & Close`.

.. note::
   Depending on the installed applications, different tabs or fields may be visible on the
   :guilabel:`Create Driver` and :guilabel:`Create Future Driver` forms.

.. _fleet/new_vehicle/general-info:

> VEHICLE

Reviewer: Headings should be sentence case

   :guilabel:`Create Driver` and :guilabel:`Create Future Driver` forms.

.. _fleet/new_vehicle/general-info:

Vehicle
-------

This section captures key physical details of a vehicle. Selecting an existing Model may auto-fill
some fields.

Fill in the following fields on the form:

- :guilabel:`Category`: Using the drop-down menu, select the vehicle category from the available
  options. If the **Inventory** app is installed, the category affects any configured :doc:`dispatch
  management system <../../inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch>`.
- :guilabel:`Order Date`: Using the calendar selector, select the date the vehicle was ordered. Keeping
  track of how long vehicles have been in the fleet can help when making decisions on expensive repairs,
  or help decide when to surrender a vehicle.
- :guilabel:`Registration Date`: Using the calendar selector, select the date the vehicle was
  registered. Many regions require proper registration, so keeping track of registration dates is important.
- :guilabel:`Cancellation Date`: Using the calendar selector, select the date the vehicle lease

>   expires, or when the vehicle is no longer part of the fleet (e.g.: sold, plates surrendered).

Reviewer 2: ```suggestion
  expires, or when the vehicle is no longer part of the fleet (e.g., sold, plates surrendered).
```

- :guilabel:`Chassis Number`: Enter the chassis number in the field. This is known in some countries
  as the :abbr:`VIN (Vehicle Identification Number)` number. Every vehicle has a unique number, so
  in the event of a theft or accident, a vehicle can be identified with this unique number.
- :guilabel:`Last Odometer`: Enter the last known odometer reading in the number field. Using the
  drop-down menu next to the number field, select whether the odometer reading is in kilometers
  :guilabel:`(km)` or miles :guilabel:`(mi)`. Keeping track of a vehicle's mileage is crucial when
  determining the value of the vehicle for both tax purposes and resale value.
- :guilabel:`Fleet Manager`: Select the fleet manager from the drop-down menu, or type in a new

>   fleet manager, and click either :guilabel:`Create` or :guilabel:`Create and Edit`.

Reviewer 2: ```suggestion
  fleet manager, and click either :guilabel:`Create` or :guilabel:`Create and edit...`.
```

> Fill in the following fields on the form:

Reviewer 2: Lines 161 - 164: 

Possible suggestion for brevity! 
This section captures key physical details of a vehicle. Selecting an existing Model may auto-fill some fields.

> - :guilabel:`Category`: Using the drop-down menu, select the vehicle category from the available

Reviewer 2: we can possibly link to the category section in your other fleet doc for more info about what this field does!

>   options. To create a new category, type in the new category name, then click :guilabel:`Create

Reviewer 2: prob don't have to detail how to do this since it's covered in another doc?

Author: Ooof, yes - but it is not merged yet! I will need to wait till it is published so I can reference it 9I think I  added a new anchor, and odn't want a merge conflict, lol). I have to "put a pin in this" for right now =/

  "category"`.

> - :guilabel:`Order Date`: Using the calendar selector, select the date the vehicle was ordered.

Reviewer 2: Great start laying out the field list! To make the section even more useful for readers, consider adding a brief “why it matters” note for each field:

- Explain Odoo’s downstream use (e.g., is *Order Date* used for other workflows, like to check for depreciation schedules? And how the *Fleet Manager* is the user who receives contract alerts).

- Highlight operational impact (why an accurate odometer reading affects maintenance reminders, why *Location* ties into understanding which site the vehicle is on, etc.).

- Where possible, include a quick-hit example (“Odoo schedules the first service 10,000 km after the Last Odometer value”).

A sentence or two per field is enough; the added context will help users see the practical value of filling everything in. Let me know if you’d like help drafting those mini-descriptions!

Author: OK, I THINK I have enough info added? I'll have you check if it's ok, though!

> - :guilabel:`Category`: using the drop-down menu, select the vehicle category from the available

Reviewer: full sentences after ":" should be capitalized

The sections below are default and appear for all vehicles, regardless of other installed
applications or localization settings.

> FISCALITY

Reviewer: Headings in sentence case :)

~~~~~~~~~

> - :guilabel:`Horsepower Taxation`: enter the amount that is taxed based on the size of the vehicle's

Reviewer: full sentences after ":" should be capitalized

  engine. This is determined by local taxes and regulations, and varies depending on the location.
  It is recommended to check with the accounting department to ensure this value is correct.
- :guilabel:`Disallowed Expenses Rate`: configure the dates and percentages of the vehicle's value
  that cannot be claimed on tax returns.

> CONTRACT

Reviewer: Sentence case :)

Author: I REALLY wish they didn't change these to all caps =D

- :guilabel:`Purchase Value`: enter the purchase price or the original value of the lease for the
  vehicle.
- :guilabel:`Residual Value`: enter the current value of the vehicle.

.. note::
   The values listed above affect the accounting department. It is recommended to check with the
   accounting department for more information and/or assistance with these values.

.. image:: new_vehicle/new-vehicle-tax.png
   :alt: The new vehicle form, showing the vehicle tax section.

Model tab
---------

If the model for the new vehicle is already configured in the database, the :guilabel:`MODEL` and
:guilabel:`ENGINE` sections are populated with the corresponding information. If the model is *not*
already in the database and the :guilabel:`Model` tab needs to be configured, :ref:`configure the
new vehicle model <fleet/add-model>`.

Check the information in the :guilabel:`Model` tab to ensure it is accurate. For example, the color
of the vehicle, or if a trailer hitch is installed, are examples of common information that may need
updating.

.. image:: new_vehicle/model-tab.png
   :alt: The new vehicle form, showing the vehicle tax section.

> Note tab

Reviewer: I would consider putting this above the image. Since it is a small section, it gets lost under the image

Fiscality
~~~~~~~~~

- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicle's
  engine. This is determined by local taxes and regulations, and varies depending on the location.
  It is recommended to check with the accounting department to ensure this value is correct.
- :guilabel:`Disallowed Expenses Rate`: Configure the dates and percentages of the vehicle-related
  costs (fuel, maintenance, depreciation, etc.) that **cannot** be deducted from the company's
  taxable income.

Contract
~~~~~~~~

- :guilabel:`First Contract Date`: Select the start date for the vehicle's first contract using the
  calendar selector. Typically this is the day the vehicle is purchased or leased.

> - :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail Price)`
>   for the vehicle at the time of purchase or lease.

Reviewer 3: Nitpick- line over 100 chars
```suggestion
- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
  Price)` for the vehicle at the time of purchase or lease.
```
